### ISSUE TITLE
Tooltips start in top left corner, then fix themselves

### ISSUE THREAD

**Issue by rwwagner90** [kaliber5/ember-bootstrap#1159]
We recently added ember-ref-modifier to our app, and it seems that having it as a dep in our app makes the tooltips render in the top left and flash real quick, then fix themselves and be in the right place. Any ideas how to fix this?

After some debugging, it seems this is only an issue when manually passing `visible`.

---

Output from `ember bootstrap:info`:

```
ember-bootstrap: ^3.1.3 -> 3.1.3
ember-cli: ~3.18.0 -> 3.18.0
bootstrap: ^4.5.0 -> 4.5.0
bootstrap-sass: n/a
ember-cli-sass: ^10.0.1 -> 10.0.1
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
importBootstrapCSS: false
bootstrapVersion: 4
importBootstrapFont: false
```

**Comment by jelhan** [comment]
Are you sure that it's related to adding ember-ref-modifier as an explicit dependency to your app? Sounds very unlikely to me. I assume your have added the latest version, aren't you? Are you able to provide a reproduction as Ember Twiddle?

**Comment by rwwagner90** [comment]
I don't seem to be able to add ember-bootstrap to a twiddle, otherwise, I would be happy to provide a reproduction. Do you have a working twiddle?

**Comment by jelhan** [comment]
`v3.1.3` should work fine. I just double checked. `v4.0.0-rc.3` is broken. But you are using v3, aren't you?

**Comment by rwwagner90** [comment]
@jelhan yes, but no idea how to get it to work with twiddle. Added it to `addons` and twiddle just kept spinning.

**Comment by jelhan** [comment]
> @jelhan yes, but no idea how to get it to work with twiddle. Added it to `addons` and twiddle just kept spinning.

Building a dependency takes a few minutes if it hasn't been build before. It reports in console if it's still building or if there was an error. But here you go: https://ember-twiddle.com/a24d34bd4cbc87124725be964d0ee7a8?openFiles=templates.application%5C.hbs%2C

**Comment by simonihmig** [comment]
That sounds related to https://github.com/kybishop/ember-popper/issues/126. Let's see what happens when updating `ember-popper`, after https://github.com/kybishop/ember-popper/pull/127 is merged. 

But having a test that reproduces this faulty behavior would certainly be nice!

**Comment by rwwagner90** [comment]
@simonihmig that does sound like the bug we were experiencing. I would call this fixed when that fix is in.

**Comment by simonihmig** [comment]
@rwwagner90 I just released v0.11.3 of ember-popper. Could you test your app with that version to confirm?

**Comment by rwwagner90** [comment]
We've refactored away from manually forcing the popover open, so I don't have a great way to test anymore. Let's call this resolved unless someone else complains. Thanks so much!
